ci/docker-db.sh

```diff
@@ -18,7 +18,7 @@ case "$DB" in
   CHECK="mysql --host $MYSQL_HOST --port $MYSQL_TCP_PORT --user root -e \q"
   ;;
 "postgres")
-  RUN_ARGS="-p 5432:5432 postgres:9.5"
+  RUN_ARGS="-e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432 postgres:9.5"
```

Comment:
This environment variable explicitly disables auth, meaning that we are not testing that we have postgres auth set up correctly. If this is required, it means that we *don't* have postgres auth set up correctly.

```suggestion
  RUN_ARGS="-p 5432:5432 postgres:9.5"
```